sycl/test/matrix/matrix-bfloat16-test-coord-basicB.cpp

```diff
@@ -0,0 +1,238 @@
+// RUN: %clangxx -fsycl -O2 -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4 %s -o %t.out
```

Comment:
these are the same tests you added to llvm-test-suite right?
If yes, keep only one here (accumulator one probably).